#world_name: brsu

plugins:
  - name: wired
    lib: libed_mulitple_hypothesis_plugin.so
    frequency: 20
    parameters:
        world_model_frame: "/map"
        output_frame: "/map"
        max_num_hypotheses: 100
        min_probability_ratio: 1e-10
        
        knowledge: 
          prior_new: 0.14
          prior_existing: 0.14 
          prior_clutter: 0.72
          object_class: 
            - name: "object"
              behavior_model: 
                 - attribute: "position"
                   model: "wire_state_estimators/PositionEstimator"
                   pnew:
                     type: "uniform"
                     dimensions: 3
                     density: 0.0001
                   pclutter:
                     type: "uniform"
                     dimensions: 3
                     density: 0.0001
                   parameters:
                     - name: "max_acceleration"
                       value: 8
                     - name: "kalman_timeout"
                       value: 1
                     - name: "fixed_pdf_cov"
                       value: 0.008

                 - attribute: "color"
                   model: "wire_state_estimators/DiscreteEstimator"
                   pnew:
                     type: "discrete"
                     domain_size: 100
                   pclutter:
                     type: "discrete"
                     domain_size: 100

                 - attribute: "class_label"
                   model: "wire_state_estimators/DiscreteEstimator"
                   pnew:
                     type: "discrete"
                     domain_size: 100
                   pclutter:
                     type: "discrete"
                     domain_size: 100

                 - attribute: "object"
                   model: "wire_state_estimators/DiscreteEstimator"
                   pnew:
                     type: "discrete"
                     domain_size: 10
                   pclutter:
                     type: "discrete"
                     domain_size: 10
